src/main/fc/fc_msp.c

```diff
@@ -142,13 +142,13 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOXSONAR, "SONAR", 22 },
     { BOXSERVO1, "SERVO1", 23 },
     { BOXSERVO2, "SERVO2", 24 },
-    { BOXSERVO3, "SERVO3", 25 },
     { BOXBLACKBOX, "BLACKBOX", 26 },
     { BOXFAILSAFE, "FAILSAFE", 27 },
     { BOXAIRMODE, "AIR MODE", 28 },
-    { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
-    { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
+    { BOX3DPOSSWITCH, "POSTIVE ONLY 3D", 29},
+    { BOX3DNEGSWITCH, "NEGITIVE ONLY 3D", 30},
```

Comment:
Sounds good to me.  There are a few thing that should be done different, such as switching when you flip the switch not when you leave deadband.

I'll add in a `switched_3d` bool to the `flight3DConfig_s` struct